common/trigger_localization/wc_character_interaction_triggers.txt

```diff
@@ -0,0 +1,7 @@
+﻿stasis_trap_already_stunned = {
+	first_not = STASIS_TRAP_ALREADY_STUNNED
+}
+
+stasis_trap_you_are_stunned = {
+	first_not = STASIS_TRAP_YOU_ARE_STUNNED
+}
```

Comment:
Check other trigger localizations.
`first` and `first_not` refer to a target in the first person (thus you) so there's no need in 2 trigger localizations.
`third` and `third_not` refer to a target in the third person (thus he, she).

Also, I propose you use this style in localization keys (instead of STASIS_TRAP_ALREADY_STUNNED):
`<Location Name>_<Type>`
Thus it's `stasis_trap_already_stunned_first`, `stasis_trap_already_stunned_first_not`, `stasis_trap_already_stunned_third`, etc.